src/common/java/net/minecraftforge/gradle/common/util/Utils.java

```diff
@@ -92,12 +92,13 @@
                                                           //manifest doesn't include sha1's so we use this for the per-version json as well.
     public static final String FORGE_MAVEN = "https://maven.minecraftforge.net/";
     public static final String MOJANG_MAVEN = "https://libraries.minecraft.net/";
+    public static final String PARCHMENT_MAVEN = "https://maven.parchmentmc.org/";
```

Comment:
Do we trust this to not go down and to be able to handle the amount of traffic that could be sent to it, or should it be mirrored (or such)?

Reply:
Linking from Dev Discord for transparancy.

We pinged @gigaherz  flagging for us being setup to be mirrored earlier this week. Since he's been our main point of contact for forge-related maven work.

We actually encourage our partners to mirror our maven to ensure we have backup copies in-case one goes down or something happens with the data on our maven. 

So I'd say forge can feel completely free to mirror our maven to feel as safe as possible.

Reply:
I forgot 👀 
I was going to mention it on the internal channel but got sidetracked.

Reply:
We will eventually mirror it, however, we need to do some development on reposilite to allow a better proxy system. For now I think we should Not explicitly auto-add these repos, but instead rely on the modder that is adding a custom channel, to add them to their repos. 

Overall the change looks like it should just be a channel -> maven coord map.